config/samples/kustomization.yaml

```diff
@@ -1,11 +1,12 @@
 ## Append samples you want in your CSV to this file as resources ##
+# NOTE(dprince): these get added to the bundle ALM examples
+# we do not have space for long examples here and want
+# to guide users to documentation instead
```

Comment:
OpenShift UI takes the first CR to prepopulate the UI, do we really want to get everything empty?

Reply:
We have always been doing this as csv-merger removed all the alm-examples. This is effectively just restoring that (old) behavior